cmake/Modules/ObsHelpers_Linux.cmake

```diff
@@ -0,0 +1,60 @@
+# Helper function to set up runtime or library targets
+function(setup_binary_target target)
+	set_target_properties(${target} PROPERTIES
+		BUILD_RPATH "${OBS_OUTPUT_DIR}/$<CONFIG>/${OBS_EXECUTABLE_DESTINATION}$<$<BOOL:LINUX_PORTABLE>:/${_ARCH_SUFFIX}bit>")
+
+	_setup_binary_target(${target})
+endfunction()
+
+# Helper function to export target to build and install tree
+# (Allows usage of `find_package(libobs)` by other build trees)
+function(export_target target)
+	_export_target(${ARGV})
+
+	if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/pkgconfig")
+		export_target_pkgconf(${target})
+	endif()
+endfunction()
+
+# Helper function to build pkgconfig file for target
+function(export_target_pkgconf target)
+	get_target_property(_TARGET_DEPENDENCIES ${target} INTERFACE_LINK_LIBRARIES)
+	get_target_property(_TARGET_DEFINITIONS ${target} INTERFACE_COMPILE_DEFINITIONS)
+	get_target_property(_TARGET_OPTIONS ${target} INTERFACE_LINK_OPTIONS)
+
+	foreach(_LIBRARY IN LISTS _TARGET_DEPENDENCIES)
+		get_target_property(_LINK_LIBRRAY ${_LIBRARY} INTERFACE_LINK_LIBRARIES)
+		get_target_property(_LINK_DEFINITIONS ${_LIBRARY} INTERFACE_COMPILE_DEFINITIONS)
+		list(APPEND _LINKED_LIBRARIES "${_LINK_LIBRRAY}")
```

Comment:
Here, and on line 26 'LIBRRAY' looks like a typo of 'LIBRARY'.

Reply:
Good catch, fixed. As both were wrong it didn't seem to have a functional impact.